packages/snowpack/src/build/build-import-proxy.ts

```diff
@@ -4,14 +4,15 @@ import {SnowpackConfig} from '../types/snowpack';
 import {getExt, URL_HAS_PROTOCOL_REGEX} from '../util';
 
 export function getMetaUrlPath(urlPath: string, isDev: boolean, config: SnowpackConfig): string {
+
   let {baseUrl, metaDir} = config.buildOptions || {};
   if (isDev) {
```

Comment:
so in build, we should want to include the baseUrl since not all sites are served from `/`. But in dev, we should hit this if block, and return before ever getting to the line you updated. So, this code looks like it should be running as expected.

Maybe isDev is false for dev mode, somehow?

Reply:
Well, it may be something i have missed here but as i understand it the baseUrl are being used for several purposes and thats why (at least for me) it gets confusing. For instance, firstly It will extend the path to the first js file (index.js) in the script tag in the index.html. But it will also extend other paths such the metaDir paths within the js files (which will in the end be resolved as relative path refereed to that index.js file). That is why the baseUrl gets double in the browser.

About the isDev block, in my eyes it could be removed because the output (with the changes i did) will now be the same. But I felt a bit unsure about what impact that removal would have so that is why i left it.